bigquery/api/Snippets/AuthenticateServiceAccount.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Google.Apis.Auth.OAuth2;
```

Comment:
You are missing the license here. Remember to change the year to 2020 if you copy it from another sample.

Reply:
Done